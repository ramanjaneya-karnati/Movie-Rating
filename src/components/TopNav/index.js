import React from 'react';
import {APP_LABELS} from '../../constants/AppLabels';

const Index = () =>{
  return (
      <div className="navbar navbar-expand-lg fixed-top navbar-dark bg-primary">
        <div className="container">
            <a href={window.location.href} className="navbar-brand">{APP_LABELS.APP_HEADER_NAME}</a>
          <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive"
                  aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
            <span className="navbar-toggler-icon"></span>
          </button>
        </div>
      </div>
  );
};

export default Index;