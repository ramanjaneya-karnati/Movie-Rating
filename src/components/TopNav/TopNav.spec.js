import {mount, shallow} from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';

import TopNav from './index';

describe('TopNav Component Testing', ()=>{
    it('renderer snapshot', () => {
        const tree = renderer.create(<TopNav/>).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('renderer without crashing', () => {
        const wrapper = shallow(<TopNav/>);
        expect(wrapper.exists()).toBe(true);
    });

});