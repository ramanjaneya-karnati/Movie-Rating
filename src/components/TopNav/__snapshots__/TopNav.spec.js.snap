// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopNav Component Testing renderer snapshot 1`] = `
<div
  className="navbar navbar-expand-lg fixed-top navbar-dark bg-primary"
>
  <div
    className="container"
  >
    <a
      className="navbar-brand"
      href="http://localhost/"
    >
      Movie Rating App
    </a>
    <button
      aria-controls="navbarResponsive"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarResponsive"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
  </div>
</div>
`;
